### PR TITLE
gateway-api: Bump to version v1.1.0

### PR DESCRIPTION
**Pull request by sayboras**
## Description

Please refer to individual commit for more details. Docs update will be added in separate PR.

Relates: #32492

### CODE REVIEW

**Review by lmb — APPROVED** [review]

**Review by brlbil — APPROVED** [review]

**Review by learnitall — APPROVED** [review]
Great work! Makes sense to me on the vendor changes.

**Comment by sayboras** [comment]
Rebase to resolve conflict with main branch

**Review by squeed — APPROVED** [review]

File: operator/pkg/model/translation/cec_translator.go [hunk 1]
```diff
@@ -133,16 +133,16 @@ func (i *cecTranslator) getBackendServices(m *model.Model) []*ciliumv2.Service {
 	return res
 }
 
-func (i *cecTranslator) getServicesWithPorts(namespace string, name string, model *model.Model) []*ciliumv2.ServiceListener {
-	// Find all the ports used in the model and build a set of them
+func (i *cecTranslator) getServicesWithPorts(namespace string, name string, m *model.Model) []*ciliumv2.ServiceListener {
+	// Find all the ports used in the m and build a set of them
```
Comment (youngnick) [inline operator/pkg/model/translation/cec_translator.go]: search and replace was too greedy here, I think.

**Review by youngnick — APPROVED** [review]
LGTM with one tiny nit.

File: operator/pkg/model/translation/cec_translator.go (hunk 1 above)
Comment (sayboras) [reply operator/pkg/model/translation/cec_translator.go]: oops thanks a lot.

**Pull request merged**